python/interpret/widget/ExplanationDashboard.py

```diff
@@ -0,0 +1,197 @@
+
+        :param explanationObject: An object that represents an explanation.
+        :type explanationObject: ExplanationMixin
+        :param learner: An object that represents a model. It is assumed that for the classification case
+            it has a method of predict_proba() returning the prediction probabilities for each
+            class and for the regression case a method of predict() returning the prediction value.
+        :type learner: object
+        :param dataset:  A matrix of feature vector examples (# examples x # features), the same sampels
+            used to build the explanationObject. Will be overwritten if set on explanation object already
+        :type dataset: numpy.array or list[][]
+        :param trueY: The true labels for the provided dataset. Will be overwritten if set on explanation object already
+        :tpye trueY: numpy.array or list[]
+        """
+        self._widget_instance = ExplanationWidget()
+        self._learner = learner
+        self._is_classifier = hasattr(learner, 'predict_proba') and learner.predict_proba is not None
+        self._dataframeColumns = None
+        if isinstance(dataset, pd.DataFrame) and hasattr(dataset, 'columns'):
+            self._dataframeColumns = dataset.columns
+        try:
+            list_dataset = self._convertToList(dataset)
+        except:
+            raise ValueError("Unsupported dataset type")
+        try:
+            y_pred = learner.predict(dataset)
+        except:
+            raise ValueError("Model does not support predict method for given dataset type")
+        try:
+            y_pred = self._convertToList(y_pred)
+        except:
+            raise ValueError("Model prediction output of unsupported type")
+        try:
+            trueY = self._convertToList(trueY)
+        except:
+            raise ValueError("True Y array of unsupported type")
+
+        dataArg = {}
+
+        row_length, feature_length = np.shape(list_dataset)
+        if row_length > 100000:
+            raise ValueError("Exceeds maximum number of rows for visualization (100000)")
+        if feature_length > 1000:
+            raise ValueError("Exceeds maximum number of features for visualization (1000)")
+        # List of explanations, key of explanation type is "explanation_type"
+        self._mli_explanations = explanationObject.data(-1)["mli"]
+        local_explanation = self._find_first_explanation(ExplanationDashboardInterface.MLI_LOCAL_EXPLANATION_KEY)
+        global_explanation = self._find_first_explanation(ExplanationDashboardInterface.MLI_GLOBAL_EXPLANATION_KEY)
+        ebm_explanation = self._find_first_explanation(ExplanationDashboardInterface.MLI_EBM_GLOBAL_EXPLANATION_KEY)
+        dataset_explanation = self._find_first_explanation(ExplanationDashboardInterface.EXPLANATION_DATASET_KEY)
+
+        dataset_x = None
+        dataset_y = None
+        predicted_y = None
+        if dataset_explanation is not None:
```

Comment:
TODO: Ignore dataset explanation until it is reliably there and in usable format